core/src/main/java/io/seata/core/store/db/AbstractDataSourceGenerator.java

```diff
@@ -60,6 +65,24 @@
 
     private static final long DEFAULT_DB_MAX_WAIT = 5000;
 
+    /**
+     * generate the data source
+     */
+    public AbstractDataSourceGenerator() {
+        this.dataSource = this.generate();
```

Comment:
recommended to implement io.seata.common.executor.Initialize

Reply:
ok